static/oas/v1/team/team.yml

```diff
@@ -0,0 +1,127 @@
+paths:
+  teams:
```

Comment:
Ryan B. said on a previous PR, "It can be either one, but /venues needs to be used if there [are] endpoints for both /venues (for Get All and Post) and /venues:id (for Get By ID and PUT)"

Reply:
I got the syntax wrong on this, Get By ID and PUT should be under `/venues/{venueId}`

Reply:
this has been resolved. It was renamed.